crates/nu-explore/src/views/record/table_widget.rs

```diff
@@ -652,10 +651,11 @@ fn truncate_list(list: &mut [NuText], width: usize) {
     }
 }
 
-fn render_shift_column(buf: &mut Buffer, x: u16, y: u16, height: u16, style: NuStyle) -> u16 {
+/// Render a column with an ellipsis in the header to indicate that there is more data than can be displayed
+fn render_overflow_column(buf: &mut Buffer, x: u16, y: u16, height: u16) -> u16 {
     let style = TextStyle {
         alignment: Alignment::Left,
-        color_style: Some(style),
+        color_style: None,
```

Comment:
It looks like you're removing `self.style.shift_line_style`, which I don't really have a problem with but if we're not rendering the `...` with a style, since it's `None` here, doesn't it show up in a different style than all the other columns, making it stand out?

Reply:
I was thinking more about this and maybe the column header styles are already be inherited from the nushell config. If that's the case, then I think I'm good here.

Reply:
`shift_line_style` wasn't being set anywhere before. So I'm pretty sure we were just using the default styling for it (which took me a while to figure out when reading through the code). Now we just always use the default style.

Reply:
ok, let's run with it. Thanks!